src/Pages/Home.jsx

```diff
@@ -5,7 +5,14 @@ import ProductTable from "../Components/ProductTable";
 
 function Home() {
   const [open, setOpen] = useState(false);
-  const [cart, updateCart] = useState([]);
+  const [cart, updateCart] = useState(() => {
+    if (localStorage.getItem("cart"))
+      return JSON.parse(localStorage.getItem("cart"));
+    else return [];
+  });
+  //   const [cart, updateCart] = useState([]);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏Great use of arrow function in useState. It's an underrated approach to defining an initial state. Great Work!